.env.sample

```diff
@@ -82,3 +82,6 @@ REVIEW_REPLY_BUFFER={"seconds":0,"minutes":0,"hours":-12,"days":0}
 
 # LINE add friend url, used for the message to guide user from LINE Notify to cofacts
 LINE_FRIEND_URL=https://line.me/R/ti/p/@cofacts
+
+# When updating url-related features such as tutorial images, change VERSION for cache busting
+VERSION=
```

Comment:
I suggest we keep `VERSION` as a variable in `tutorial.js` because
- If we change image in the future, it's natural that we go to tutorial.js and make changes there
- Since image updates are under source control, so should its `VERSION`. The change of image file and the change of `VERSION` variable should be updated together in a PR.